src/sentry_plugins/hipchat_ac/templates/sentry_hipchat_ac/configure.html

```diff
@@ -1,5 +1,38 @@
 {% extends "sentry_hipchat_ac/configure_layout.html" %}
 {% block body %}
+
+<script>
+// open sentry login in another window and then receive a message of whether user is logged in. If user is logged in, refresh the page, so that user can continue through configuration flow
+  function extractOrigin() {
+    var pathArray = location.href.split( '/' );
+    var protocol = pathArray[0];
+    var host = pathArray[2];
+    var origin = protocol + '//' + host;
+    return origin;
+  }
+
+  var popup;
+
+  function openLogin() {
+    var top = screen.height / 2 - 350
+    var left = screen.width / 2 - 300
+    popup = window.open(
```

Comment:
Initially I thought I might be sending a message with postMessage, but it does not need to be bound. I'll make this change.